src/foam/core/reflow/PropertyRichListView.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core.reflow',
+  name: 'PropertyRichListView',
+  extends: 'foam.core.reflow.PropertyListView',
+
+  properties: [
+    {
+      name: 'choice',
+      view: function(_, X) { return { class: 'foam.core.reflow.PropertyRichChoiceView', of: X.data.of } },
```

Comment:
```suggestion
      view: function(_, X) { 
      if ( !  X.data.of ) return; // Maybe throw an error here
      let arr = X.data.of.getAxiomsByClass(foam.lang.Property).map(p => { 
        return  { id: p, name: p.name };
      });
      return { 
        class: 'foam.u2.RichChoiceView', 
        of: X.data.of, 
        sections: [ heading: 'Properties',
        dao: foam.dao.ArrayDAO.create({ array: arr },X) ]
      } 
      },
```

This should be able to fully replace the `PropertyRichChoiceView` assuming i didnt make syntax errors